pkg/nfd-master/nfd-master.go

```diff
@@ -50,6 +50,11 @@ import (
 	"sigs.k8s.io/node-feature-discovery/pkg/version"
 )
 
+const (
+	defaultDenyNs    = "kubernetes.io"
+	defaultDenySubNs = "*.kubernetes.io"
```

Comment:
You want to drop the asterisk (`*`) from here. Also suggest a slight renaming

```suggestion
	defaultDenyAnnotationNsSuffix = ".kubernetes.io"
```

Reply:
`defaultDenySubNs = ".kubernetes.io"`